Pokemon Showdown Enhanced Tooltips/js/showPokemonTooltip.js

```diff
@@ -436,6 +436,29 @@ ShowdownEnhancedTooltip.BattleTypeChart = {
 	},
 };
 
+ShowdownEnhancedTooltip.getStatbarHTML = function getStatbarHTML(pokemon) {
+	let buf = '<div class="statbar' + (this.siden ? ' lstatbar' : ' rstatbar') + '" style="display: none">';
+	const ignoreNick = this.siden && (this.scene.battle.ignoreOpponent || this.scene.battle.ignoreNicks);
+	buf += `<strong><a href="https://www.smogon.com/dex/sm/pokemon/${pokemon.id}/" target="_blank" style="color: #222222; text-decoration: none;">${BattleLog.escapeHTML(ignoreNick ? pokemon.species : pokemon.name)}</a>`;
+	let gender = pokemon.gender;
+	if (gender) {
+		buf += ` <img src="${Dex.resourcePrefix}fx/gender-${gender.toLowerCase()}.png" alt="${gender}" />`;
+	}
+	buf += (pokemon.level === 100 ? `` : ` <small>L${pokemon.level}</small>`);
+
+	let symbol = '';
+	if (pokemon.species.indexOf('-Mega') >= 0) symbol = 'mega';
+	else if (pokemon.species === 'Kyogre-Primal') symbol = 'alpha';
+	else if (pokemon.species === 'Groudon-Primal') symbol = 'omega';
+	if (symbol) {
+		buf += ` <img src="${Dex.resourcePrefix}sprites/misc/${symbol}.png" alt="${symbol}" style="vertical-align:text-bottom;" />`;
+	}
+
+	buf += `</strong><div class="hpbar"><div class="hptext"></div><div class="hptextborder"></div><div class="prevhp"><div class="hp"></div></div><div class="status"></div>`;
+	buf += `</div>`;
+	return buf;
+}
+
```

Comment:
Overrides the `getStatbarHTML` method in [the original Showdown Client](https://github.com/Zarel/Pokemon-Showdown-Client/blob/master/src/battle-animations.ts).

This makes the pokemon's name "clickable" with a link to their entry in smogon.